Code/GeorgiaLibrarySystem-/Tests/Unit tests/UnitTest1.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using GtlService;
+using NUnit.Framework;
+
+namespace Tests
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void Test1()
+        {
+            int c = 4;
```

Comment:
maybe you could have added text indicating where it is test setup,results and assertion (forgot if these are the propper names :) )